### PR TITLE
11 sorting articles

### PR DESCRIPTION
**Pull request by IsmailMohamed125**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -113,6 +113,57 @@ describe("Articles Endpoint", () => {
...
+      });
+      test("GET:200 - Responds with an array containing array objects sorted by the query parameter in the order direction", () => {
+        return request(app)
+          .get("/api/articles?sort_by=votes&order=asc")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body.articles;
+            expect(articles).toBeSortedBy("votes");
+          });
+      });
```
Comment (nc-jo-watson) [inline __tests__/app.test.js:165]: Nice to see this test included 👍

File: models/articles.js [hunk 2]
```diff
@@ -11,7 +29,7 @@ function selectArticles() {
           JOIN comments
           on articles.article_id = comments.article_id 
           GROUP BY articles.article_id
-          ORDER BY articles.created_at DESC`
+          ORDER BY articles.${sort_by} ${order.toUpperCase()}`
```
Comment (nc-jo-watson) [inline models/articles.js:32]: This leaves your server open to SQL injection. What can you use instead of string interpolation to create this query ?

**Review by nc-jo-watson — COMMENTED** [review]
Nice Ismail! Good use of green listing in your model. See my comment about your SQL query though. You've done it in your `selectArticle` model 👀

File: models/articles.js (hunk 2 above)
Comment (IsmailMohamed125) [reply models/articles.js:32]: Hi Jo, I thought we couldn't use parameterized queries ($1) with ORDER BY statements and green listing would be sufficient although could use pg format to sanitise the data.

**Pull request merged**